test/integration/read-write-concern/write_concern.test.ts

```diff
@@ -130,9 +136,10 @@ describe('Write Concern', function () {
           await col.createIndex({ b: -1 });
           await col.createIndex({ a: 1, b: -1 });
 
+          await sleep(1000);
```

Comment:
Why is this sleep necessary? We are just asserting that `listIndexes` does not error. If we wanted to ensure it was returning the correct indexes I would explicitly set the write concern to 'majority' when creating the indexes.